src/driver/args.c

```diff
@@ -110,100 +105,116 @@ void yf_parse_args(int argc, char ** argv, struct yf_args * args) {
             continue;
         }
 
-        if (STREQ(arg, "-h") || STREQ(arg, "--help")) {
-            yf_check_action(args, YF_HELP);
-            continue;
-        }
+        if (arg[0] == '-') {
+            ++arg;
+            if (!arg[1]) {
+                if (arg[0] == 'h' || arg[0] == '?') {
+                    yf_check_action(args, YF_HELP);
+                    continue;
+                }
 
-        if (STREQ(arg, "-v") || STREQ(arg, "--version")) {
-            yf_check_action(args, YF_VERSION);
-            continue;
-        }
+                if (arg[0] == 'v') {
+                    yf_check_action(args, YF_VERSION);
+                    continue;
+                }
+            }
 
-        if (STREQ(arg, "-native-compiler")) {
-            want_compiler_name = true;
-            /* Make sure there actually is a compiler to parse after */
-            if (i + 1 == argc) {
-                yf_set_error(args);
-                return;
+            if (arg[0] == '-')
+                ++arg;
+
+            if (STREQ(arg, "help")) {
+                yf_check_action(args, YF_HELP);
+                continue;
             }
-            continue;
-        }
 
-        if (STREQ(arg, "-compiler-type")) {
-            want_compiler_type = true;
-            if (i + 1 == argc) {
-                yf_set_error(args);
-                return;
+            if (STREQ(arg, "version")) {
```

Comment:
The string is `--version`, not `-version`.

Reply:
I made it so that long options can be specified with either one dash or two.